subgraphs/aave-v2/src/common/constants.ts

```diff
@@ -0,0 +1,159 @@
+export const ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
+
+export const UNISWAP_V2_FACTORY = "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f";
+
+export const WETH_ADDRESS = "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2";
+export const USDC_WETH_PAIR = "0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc"; // created 10008355
+export const DAI_WETH_PAIR = "0xa478c2975ab1ea89e8196811f51a7b7ade33eb11"; // created block 10042267
+export const USDT_WETH_PAIR = "0x0d4a11d5eeaac28ec3f61d100daf4d40471f1852"; // created block 10093341
+
+////////////////////////
+///// Type Helpers /////
+////////////////////////
+
+export const DEFAULT_DECIMALS = 18;
+
+export const USDC_DECIMALS = 6;
+export const USDC_DENOMINATOR = BigDecimal.fromString("1000000");
+
+export const BIGINT_ZERO = BigInt.fromI32(0);
+export const BIGINT_ONE = BigInt.fromI32(1);
+export const BIGINT_TWO = BigInt.fromI32(2);
+export const BIGINT_THOUSAND = BigInt.fromI32(1000);
+export const BIGINT_MAX = BigInt.fromString(
+  "115792089237316195423570985008687907853269984665640564039457584007913129639935",
+);
+
+export const INT_NEGATIVE_ONE = -1 as i32;
+export const INT_ZERO = 0 as i32;
+export const INT_ONE = 1 as i32;
+export const INT_TWO = 2 as i32;
+export const INT_FOUR = 4 as i32;
+
+export const BIGDECIMAL_ZERO = new BigDecimal(BIGINT_ZERO);
+export const BIGDECIMAL_ONE = new BigDecimal(BIGINT_ONE);
+export const BIGDECIMAL_TWO = new BigDecimal(BIGINT_TWO);
+
+export const MAX_UINT = BigInt.fromI32(2).times(BigInt.fromI32(255));
+
+/////////////////////
+///// Date/Time /////
+/////////////////////
+
+export const SECONDS_PER_DAY = 60 * 60 * 24; // 86400
+export const MS_PER_DAY = new BigDecimal(BigInt.fromI32(24 * 60 * 60 * 1000));
+export const DAYS_PER_YEAR = new BigDecimal(BigInt.fromI32(365));
+export const MS_PER_YEAR = DAYS_PER_YEAR.times(new BigDecimal(BigInt.fromI32(24 * 60 * 60 * 1000)));
+
+/////////////////////////////
+///// Protocol Specific /////
+/////////////////////////////
+
+// This is the LendingPoolAddressesProviderRegistry contract address
+export const PROTOCOL_ADDRESS = '0x52D306e36E3B6B02c153d0266ff0f85d18BCD413';
+
+export const USDC_TOKEN_ADDRESS = '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48';
+
+export const PRICE_ORACLE_ADDRESS = '0xa50ba011c48153de246e5192c8f9258a2ba79ca9';
+
+export const INCENTIVE_CONTROLLER_ADDRESS = '0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5';
+
+// For the Ethereum Mainnet, this is stkAAVE. Will need to set this in a config file depending on whether the network is Matic or Avax
+export const REWARD_TOKEN_ADDRESS = '0x4da27a545c0c5b758a6ba100e3a049001de870f5';
+
+export const LENDING_POOL_ADDRESS = '0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9';
+
+export const SUBGRAPH_VERSION = '1.1.26';
+export const SCHEMA_VERSION = '1.1.0';
+export const PROTOCOL_NAME = 'Aave-v2';
+export const PROTOCOL_SLUG = 'aave-v2';
+export const TOTAL_REVENUE_USD = BIGDECIMAL_ZERO;
```

Comment:
You don't need to save the `BIGDECIMAL_ZERO`s as constants. TotalRevenueUSD is not a constant that's equal to `BIGDECIMAL_ZERO`, so you should just initialize the variable as `BIGDECIMAL_ZERO`